openssl-dynamic/src/main/c/ocsp.c

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "ocsp.h"
+#include "tcn.h"
+
+#include <stdio.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
The story goes like this: We're currently using a separate JNI library to implement OCSP and we link it against tcnative. Our security team asked to built it with strict validation and we asked them to set it up. I assume `ssl_private.h` didn't pass strict validation and that's why they added it.

Now, I just copy & pasted it here without putting much thought into it. It's not needed in this context and we can remove it and do a separate PR for changing compiler flags.